src/Kiota.Builder/Writers/Shell/ShellCodeMethodWriter.cs

```diff
@@ -156,6 +147,40 @@ private void WriteExecutableCommand(CodeMethod codeElement, RequestParams reques
             writer.WriteLine("return command;");
         }
 
+        private void AddCustomCommandOptions(LanguageWriter writer, ref List<string> availableOptions, ref List<string> paramTypes, ref List<string> paramNames, string returnType, bool isHandlerVoid)
+        {
+            if (conventions.StreamTypeName.Equals(returnType, StringComparison.OrdinalIgnoreCase))
+            {
+                var fileOptionName = "fileOption";
```

Comment:
should we also move `fileOptionName` and `outputOptionName` up to be `private const string` as the other strings used for interpolation?